db/migrate/20180827225542_create_webauthn_configurations_table.rb

```diff
@@ -0,0 +1,12 @@
+class CreateWebauthnConfigurationsTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :webauthn_configurations do |t|
```

Comment:
Something Jon mentioned the other day that would be good to have is a name for the configuration. I haven't added that to the phone configurations table yet, but plan to "soon."